vnext/ReactUWP/Views/ControlViewManager.cpp

```diff
@@ -64,8 +64,14 @@ void ControlViewManager::UpdateProperties(ShadowNodeBase *nodeToUpdate, const fo
       } else if (propertyName == "tabIndex") {
         if (propertyValue.isNumber()) {
           auto tabIndex = propertyValue.asDouble();
-          if (tabIndex == static_cast<int32_t>(tabIndex))
-            control.TabIndex(static_cast<int32_t>(tabIndex));
+          if (tabIndex == static_cast<int32_t>(tabIndex)) {
+            if (tabIndex < 0) {
+              control.IsTabStop(false);
```

Comment:
>control.IsTabStop(false); [](start = 14, length = 25)

In addition to setting IsTabStop = false, consider clearing the TabIndex property.

Reply:
This makes sense. Now clearing the property and tested that everything continues to work.

---
In reply to: [385377044](https://github.com/microsoft/react-native-windows/pull/4180#discussion_r385377044) [](ancestors = 385377044)